### PR TITLE
Create automated follow tiltados

### PR DESCRIPTION
**Pull request by helderfalcao**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,20 +28,86 @@ app.get('/:name/:days/:record', (req, res) => {
     })
 })
 
+app.get('/:name/:days/:record', (req, res) => {
```
Comment (DennyLoko) [inline index.js]: Why did you duplicate this route?

File: index.js [hunk 2]
```diff
@@ -25,20 +28,86 @@ app.get('/:name/:days/:record', (req, res) => {
...
+            renderLastTilt(res, savedTiltado.name, daysDiff, savedTiltado.record < daysDiff ? daysDiff : savedTiltado.record);
+        } else {
+            res.send("usuário Inválido")
+        }
+    })
+    
+})
+
+
+app.get('/source/:name', (req, res) => {
```
Comment (DennyLoko) [inline index.js]: Why did you create this route? It doesn't have all the required parameters to render the correct image and it doesn't query the database to get the user's data, so I'm a little bit lost about it...

**Review by DennyLoko — CHANGES_REQUESTED** [review]

**Comment by DennyLoko** [comment]
Hey @helderfalcao, thanks for taking the time to submit a PR to the project!

I would like to suggest some changes... I think it's not a good idea to use the user name as the primary key on Mongo because it would be too easy to have conflicts, so I would like to suggest you to change it to be a generated ID.

Also, I think we will need to change the routes to be:
- `POST /u` receiving the user name and returning a JSON response with the user ID
- `GET /i/u/:id` to return the image with the data from the database
- `GET /u/:id` to display the internal page with the data from the database
- `GET /u/:id/tiltou` to update the `lastTilt` timestamp and displaying the internal page

I guess with those changes we will be able to handle more properly the user registration. What do you think?

Thanks!

File: index.js (hunk 1 above)
Comment (helderfalcao) [reply index.js]: No reason, just a copy and paste error

File: index.js (hunk 2 above)
Comment (helderfalcao) [reply index.js]: Just an error during the copy and paste, the route was already removed.

**Review by DennyLoko — APPROVED** [review]

**Pull request merged**